momepy/dimension.py

```diff
@@ -898,6 +875,8 @@ class PerimeterWall:
 
     Notes
     -----
+    The ``spatial_weights`` keyword argument should be based on index, not unique ID.
```

Comment:
It is actually based on a position, not the index either.... We should probably clarify that and make it consistent across momepy. It is a mess right now (partilly caused by those inconsistencies in libpysal).